src/components/carbon-provider/carbon-provider.test.tsx

```diff
@@ -0,0 +1,38 @@
+import React from "react";
+import { render, screen } from "@testing-library/react";
+import CarbonProvider from "../carbon-provider";
+import Button from "../button";
+
+describe("The CabronProvider component", () => {
+  test("renders with aria-hidden false if no ariaHidden prop is specified", () => {
+    render(
+      <CarbonProvider>
+        <Button buttonType="primary">Button</Button>
+      </CarbonProvider>
+    );
+
+    const button = screen.getByRole("button", {
+      name: "Button",
+    });
+
+    // eslint-disable-next-line testing-library/no-node-access
+    const parent = button.closest("div");
```

Comment:
comment: I'd prefer to add a `data-role` to the div and locate it with `getByTestId`